quartz/util/socialImage2.tsx

```diff
@@ -0,0 +1,80 @@
+import { SatoriOptions } from "satori/wasm"
```

Comment:
wait why there are two socialImage?

Reply:
Ah right I think that is a leftover from when I changed it to the new template, been a while but I think we talked about leaving another one in as reference but might be better off just putting it in docs or just removing in general?

Reply:
yeah let's do what you suggest here (put one in docs)